data_model/changelogs/0005/0005_fix_img_size.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE signalo_db.vl_official_sign SET img_width = 72 WHERE id = '4.25b';
```

Comment:
this file needs to be added here https://github.com/opengisch/signalo/blob/master/data_model/setup.sh#L77